data_uploader/netbox_api/netbox_dcim.py

```diff
@@ -0,0 +1,104 @@
+from data_uploader.netbox_api.netbox_connection import NetboxApi
+
+
+class NetboxDcim():
+    def __init__(self, url, token, cert=None):
+        self.url = url
+        self.token = token
+        self. cert = cert
+            #'= NetboxApi.api_object(url, token, cert)
+
+        self._netbox_api = NetboxApi.api_object(url, token, cert)
+
+    def get_device(self, hostname: str):
```

Comment:
Can we add return types too:

`def get_device(self, hostname: str) -> Union[Dict[Record], None]:`

(That is assuming there's a `Record` class or similar in Python. If not you can just use `Union[Dict, None]` to show it will either return a dict or None